pubspec.yaml

```
@@ -53,15 +53,14 @@ dev_dependencies:
 
 # The following section is specific to Flutter.
 flutter:
-
   # The following line ensures that the Material Icons font is
   # included with your application, so that you can use the icons in
   # the material Icons class.
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  assets:
-     - assets/
+  # assets:
```

```suggestion
  # assets:
```여기도 다르네요. 뭔가 처음에 꼬인 부분이 있었나봅니다. ^^